tl-its-umich-edu/my-learning-analytics
# #1344 use user name value from LTI launch data

#1344  omit the user name retrieval from cron job

Test Plan:

1. make sure the dashboard/migrations/0024_remove_user_name.py migration file is applied
2. run cron job; verify the sis_name field in User table contains only null value
4. launch the tool from Canvas; notice the User.sis_name field is updated upon LTI launch for this user; 
5. verify user can use the MyLA tool for this course
6. verify user can click on the top right profile button and can see user name in "Sign in as FULL NAME"
